test/src/test/resources/fixtures/skip-include-fields/handles-skip-directive-on-single-top-level-field.yml

```diff
@@ -0,0 +1,31 @@
+name: handles skip directive on single top level field
+enabled:
+  current: false # Not fixing
+  nextgen: true
+overallSchema:
+  service: |
+    service service {
```

Comment:
nit: do we need this `service service {` ?

Reply:
Nope, removed for all skip/include tests.